vmdb/app/views/ops/_rbac_tag_box.html.haml

```diff
@@ -3,7 +3,7 @@
   %table.style1
     %tr
       %td.key
-        = session[:customer_name]
+        = current_brand.customer_name
```

Comment:
Would it make sense to just have a customer_name method in the application_helper/application_controller that delegates to `current_brand` (or `current_tenant`)?  Then this wouldn't have to be explicit all over the code.  Not 100% sure.